include/open62541/server_pubsub.h

```diff
@@ -714,6 +766,11 @@ UA_Server_setReaderGroupOperational(UA_Server *server, const UA_NodeId readerGro
 UA_StatusCode UA_EXPORT
 UA_Server_setReaderGroupDisabled(UA_Server *server, const UA_NodeId readerGroupId);
 
+#ifdef UA_ENABLE_PUBSUB_MONITORING
+UA_StatusCode UA_EXPORT
+UA_Server_setPubSubMonitoringCallbacks(UA_Server *server, const UA_PubSubMonitoringInterface * const monitoringInterface);
+#endif /* UA_ENABLE_PUBSUB_MONITORING */
```

Comment:
Not sure if this API function is needed. The user has access to server config and the function only sets the new PTR.

Reply:
yes you are right. I have removed the function